installing/installing-nvidia-gpu-aws.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
```

Comment:
It looks like he have a duplicate assembly for the enabling GPU during installation. This file and installing/installing_aws/installing-nvidia-gpu-aws.adoc.

We might only need one.